src/fragment-components/a-tag.tsx

```diff
@@ -0,0 +1,192 @@
+	const typeItems = [
+		{id: 'gray', text: 'Gray'},
+		{id: 'red', text: 'Red'},
+		{id: 'magenta', text: 'Magenta'},
+		{id: 'purple', text: 'Purple'},
+		{id: 'blue', text: 'Blue'},
+		{id: 'cyan', text: 'Cyan'},
+		{id: 'teal', text: 'Teal'},
+		{id: 'green', text: 'Green'},
+		{id: 'cool-gray', text: 'Cool gray'},
+		{id: 'warm-gray', text: 'Warm gray'},
+		{id: 'high-contrast', text: 'High contrast'},
+		{id: 'outline', text: 'Outline'},
+	];
+
+	const sizeItems = [
+		{id: 'md', text: 'Medium'},
+		{id: 'sm', text: 'Small'}
+	];
+
+	return <>
+		<TextInput
+			value={selectedComponent.title}
+			labelText='Title'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					title: event.currentTarget.value
+				});
+			}}
+		/>
+
+		<Dropdown
+			label='Type'
+			titleText='Type'
+			items={typeItems}
+			initialSelectedItem={typeItems.find(item => item.id === selectedComponent.kind)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				kind: event.selectedItem.id
+			})}
+		/>
+
+		<Dropdown
+			label='Size'
+			titleText='Size'
+			items={sizeItems}
+			initialSelectedItem={typeItems.find(item => item.id === selectedComponent.size)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				size: event.selectedItem.id
+			})}
+		/>
+
+		<Checkbox
+			labelText='Is filter'
+			id='filter'
+			checked={selectedComponent.filter}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					filter: checked
+				});
+			}}
+		/>
+
+		<Checkbox
+			labelText='Disabled'
+			id='disabled'
+			checked={selectedComponent.disabled}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					disabled: checked
+				});
+			}}
+		/>
+
+		<ComponentCssClassSelector componentObj={selectedComponent} setComponent={setComponent} />
+	</>
+};
+
+export const ATag = ({
+	children,
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		{...rest}>
+			<Tag
+			type={componentObj.kind}
+			disabled={componentObj.disabled}
+			filter={componentObj.filter}
+			className={componentObj.cssClasses?.map((cc: any) => cc.id).join(' ')}>
+				{children}
+			</Tag>
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: ATag,
+	styleUI: ATagStyleUI,
+	render: ({ componentObj, select, remove, selected }) => <ATag
+		componentObj={componentObj}
+		select={select}
+		remove={remove}
+		selected={selected}>
+			{componentObj.title}
+	</ATag>,
+	keywords: ['tag'],
+	name: 'Tag',
+	defaultComponentObj: {
+		type: 'tag',
+		// This field is `type` in the `Tag` component
+		kind: 'gray',
+		size: 'md',
+		filter: false,
+		disabled: false,
+		title: 'Tag'
+	},
+	image,
+	codeExport: {
+		angular: {
```

Comment:
Getting
```
Template parse errors:
Can't bind to 'disabled' since it isn't a known property of 'ibm-tag'.
1. If 'ibm-tag' is an Angular component and it has 'disabled' input, then verify that it is part of this module.
2. If 'ibm-tag' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.
3. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component. ("<ibm-tag [type]="tag2Type" [title]="tag2Title" size="sm" [ERROR ->][disabled]="false">
	Taggfgkh
</ibm-tag>
```
for angular export

Reply:
Turns out disabled doesn't exist on regular tag in angular, only in filter tag - and even in that case doesn't seem to do anything.
Perhaps open issues for it in cca and here (as a dependant of it) and export disable conditionally for the time being.